easybuild/easyblocks/generic/rpackage.py

```diff
@@ -132,12 +133,8 @@ def make_cmdline_cmd(self, prefix=None):
         else:
             prefix = ''
 
-        if self.cfg['unpack_sources']:
-            loc = self.start_dir
-        elif self.patches:
-            loc = self.ext_dir
-        else:
-            loc = self.ext_src
+        loc = self.start_dir or self.ext_dir or self.ext_src
```

Comment:
How confident are you that this code cleanup is OK to do? It's not exactly equivalent to what was there...

Reply:
Very. start_dir is set to the path to the extracted sources when used as a EasyBlock. ext_dir is set for use as an Extension. ext_src is just a fallback if neither is set.

The basic idea is to always extract, see https://github.com/easybuilders/easybuild-easyblocks/pull/1938#issuecomment-582362995

Reply:
OK, makes sense.